cont3xt/integration.js

```diff
@@ -560,29 +584,29 @@ class Integration {
    */
   static async apiSearch (req, res, next) {
     if (!ArkimeUtil.isString(req.body.query)) {
-      return res.send({ success: false, text: 'Missing query' });
+      return res.send({ purpose: 'init', success: false, text: 'Missing query' });
```

Comment:
why have purpose:init when success is false?  Seems like it should either just have success or just have purpose:error or something? I vote for if there is an error no purpose is needed.

Reply:
I was thinking similar, I think I'll change this to purpose: 'error', without success=false; then I can also remove success: true, from purpose: 'init', since it will be implied.